src/erlcloud_ses.erl

```diff
@@ -1128,6 +1201,12 @@ encode_opts([{reply_to_addresses, List} | T], Acc) ->
 encode_opts([{return_path, ReturnPath} | T], Acc) ->
     encode_opts(T, [{"ReturnPath", ReturnPath} | Acc]).
 
+encode_raw_opts([], Acc) ->
+    Acc;
+encode_raw_opts([{source, Source} | T], Acc) when is_list(Source); is_binary(Source) ->
+    encode_raw_opts(T, [{"Source", Source} | Acc]);
+encode_raw_opts([{destinations, Destination} | T], Acc) ->
+    encode_raw_opts(T, encode_destinations(Destination, Acc)).
```

Comment:
Why do you pass Acc into `encode_destinations(Destination, Acc)`?

Reply:
NVM, I see the point there.

Reply:
So the values can be added to the current `Acc` but I will rework this to have the values returned from `encode_destinations` to be the head of the `Acc` and return `[encode_destinations(Destination) | Acc]`

Reply:
I can do this but this breaks from the style of the module where functions always are passes the `Acc`
Example [here](https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_ses.erl#L1012-L1019)
```erlang
encode_params([], Acc) ->
    lists:reverse(Acc);
encode_params([{body, Body} | T], Acc) ->
    encode_params(T, encode_body(Body, Acc));
encode_params([{destination, Destination} | T], Acc) ->
    encode_params(T, encode_destination(Destination, Acc));
encode_params([{email_address, EmailAddress} | T], Acc) when is_list(EmailAddress); is_binary(EmailAddress) ->
    encode_params(T, [{"EmailAddress", EmailAddress} | Acc]);
```

When a function is called, the current `Acc` is currently passed in vs when adding a sing value to the `Acc`